### ISSUE TITLE
switch is not accessible via keyboard navigation

### ISSUE THREAD

**Issue by catherine-jones** [thoughtbot/refills#397]
the switch refill is not accessible via keyboard navigation nor are you able to toggle the switch via the keyboard. I would be interested in hearing the communities thoughts around refills and accessibility.

**Comment by tysongach** [comment]
Yes, this is a huge bummer.

We are currently re-building Refills from a ground-up with a focus on a11y. An early WIP can be found here: http://refills.netlify.com/. Stay tuned!
